### PR TITLE
Fix base type of UIErrorEventArgs

### PR DESCRIPTION
**Pull request by danroth27**
I don't believe `UIErrorEventArgs` should derive from `UIProgressEventArgs`. Switching base type to `UIEventArgs` instead.

### CODE REVIEW

**Comment by galvesribeiro** [comment]
Yeah that was a mistake, sorry about that. Just confirmed https://developer.mozilla.org/en-US/docs/Web/API/ErrorEvent

Thanks for the fix!

**Comment by galvesribeiro** [comment]
In fact, I don't remember why I didn't added the `message` field at least there...

File: src/Microsoft.AspNetCore.Blazor/UIEventArgs.cs [hunk 1]
```diff
@@ -168,7 +168,7 @@ public class UIDataTransferItem
     /// <summary>
     /// Supplies information about an error event that is being raised.
     /// </summary>
-    public class UIErrorEventArgs : UIProgressEventArgs
```
Comment (rynowak) [inline src/Microsoft.AspNetCore.Blazor/UIEventArgs.cs]: This should have some properties https://developer.mozilla.org/en-US/docs/Web/API/ErrorEvent

File: src/Microsoft.AspNetCore.Blazor.Browser.JS/src/Rendering/EventForDotNet.ts [hunk 2]
```diff
@@ -248,7 +248,7 @@ interface UIDataTransferItem {
   type: string;
 }
 
-interface UIErrorEventArgs extends UIProgressEventArgs {
```
Comment (rynowak) [inline src/Microsoft.AspNetCore.Blazor.Browser.JS/src/Rendering/EventForDotNet.ts]: https://developer.mozilla.org/en-US/docs/Web/API/ErrorEvent

**Comment by rynowak** [comment]
@SteveSandersonMS - quick review please 😆

**Comment by SteveSandersonMS** [comment]
@rynowak It looks like we're still not using the `UIErrorEventArgs` type. Shouldn't we add something under line 55 in `EventForDotNet.ts` to make it parse the event object as a `UIErrorEventArgs`?

**Comment by galvesribeiro** [comment]
It should have an even handler...

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by rynowak** [comment]
> It should have an even handler...

Can you explain that in more detail? I'm not sure what that means

**Comment by rynowak** [comment]
@SteveSandersonMS updated

**Pull request merged**